packages/remix-cloudflare/index.ts

```diff
@@ -31,16 +31,19 @@ export type {
   DataFunctionArgs,
   EntryContext,
   ErrorBoundaryComponent,
+  ErrorBoundaryProps,
```

Comment:
@mbrowne I think we should add these to the `replace-remix-imports` migration too.

https://github.com/remix-run/remix/blob/15bf240cddf871494522e4c917fa17400b89dda5/packages/remix-dev/cli/migrate/run/transforms/replace-remix-imports/jscodeshift-transform/map-normalized-imports/package-exports.ts#L27

Make sure to rebase on latest `dev` to be able to access this file

Reply:
I'm happy to add this, but I just wanted to double-check that it's necessary? It says `runtime` exports and these are just types that aren't needed at run-time. Please confirm and then I can add them if needed.

Reply:
Just checking in on this...should I go ahead and add these to the `replace-remix-imports` migration? ...given that these are types used only at compile-time